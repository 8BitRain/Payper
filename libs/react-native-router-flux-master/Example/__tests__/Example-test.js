import 'react-native';
import React from 'react';

import Example from '../Example';
jest.unmock('Image');

import renderer from 'react-test-renderer';

describe('Example', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Example />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
