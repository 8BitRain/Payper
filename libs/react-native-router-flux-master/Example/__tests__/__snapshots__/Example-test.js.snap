exports[`Example renders correctly 1`] = `
<View
  onLayout={[Function bound _onLayout]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "flex": 1
      },
      undefined
    ]
  }>
  <View
    style={
      Object {
        "flex": 1
      }
    }>
    <View
      onMoveShouldSetResponder={[Function onMoveShouldSetResponder]}
      onMoveShouldSetResponderCapture={[Function onMoveShouldSetResponderCapture]}
      onResponderEnd={[Function onResponderEnd]}
      onResponderGrant={[Function onResponderGrant]}
      onResponderMove={[Function onResponderMove]}
      onResponderReject={[Function onResponderReject]}
      onResponderRelease={[Function onResponderRelease]}
      onResponderStart={[Function onResponderStart]}
      onResponderTerminate={[Function onResponderTerminate]}
      onResponderTerminationRequest={[Function onResponderTerminationRequest]}
      onStartShouldSetResponder={[Function onStartShouldSetResponder]}
      onStartShouldSetResponderCapture={[Function onStartShouldSetResponderCapture]}
      pointerEvents="auto"
      style={
        Object {
          "backgroundColor": "#fff",
          "bottom": 0,
          "flex": 1,
          "left": 0,
          "opacity": 1,
          "position": "absolute",
          "right": 0,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 0
            },
            Object {
              "translateY": 0
            }
          ]
        }
      }>
      <View
        style={
          Array [
            Object {
              "flex": 1
            },
            undefined
          ]
        }>
        <View
          style={
            Object {
              "flex": 1
            }
          }>
          <View
            onLayout={[Function bound _onLayout]}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "flex": 1
                },
                undefined
              ]
            }>
            <View
              style={
                Object {
                  "flex": 1
                }
              }>
              <View
                onMoveShouldSetResponder={[Function onMoveShouldSetResponder]}
                onMoveShouldSetResponderCapture={[Function onMoveShouldSetResponderCapture]}
                onResponderEnd={[Function onResponderEnd]}
                onResponderGrant={[Function onResponderGrant]}
                onResponderMove={[Function onResponderMove]}
                onResponderReject={[Function onResponderReject]}
                onResponderRelease={[Function onResponderRelease]}
                onResponderStart={[Function onResponderStart]}
                onResponderTerminate={[Function onResponderTerminate]}
                onResponderTerminationRequest={[Function onResponderTerminationRequest]}
                onStartShouldSetResponder={[Function onStartShouldSetResponder]}
                onStartShouldSetResponderCapture={[Function onStartShouldSetResponderCapture]}
                pointerEvents="auto"
                style={
                  Object {
                    "backgroundColor": "#fff",
                    "bottom": 0,
                    "flex": 1,
                    "left": 0,
                    "marginBottom": 0,
                    "marginTop": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "shadowColor": null,
                    "shadowOffset": null,
                    "shadowOpacity": null,
                    "shadowRadius": null,
                    "top": 0,
                    "transform": Array [
                      Object {
                        "translateX": 0
                      },
                      Object {
                        "translateY": 0
                      }
                    ]
                  }
                }>
                <View
                  style={
                    Array [
                      Object {
                        "flex": 1
                      },
                      undefined
                    ]
                  }>
                  <View
                    component={[Function Launch]}
                    getSceneStyle={[Function getSceneStyle]}
                    index={0}
                    initial={true}
                    name="launch"
                    navigationState={
                      Object {
                        "component": [Function Launch],
                        "getSceneStyle": [Function getSceneStyle],
                        "index": 0,
                        "initial": true,
                        "key": "0_launch",
                        "name": "launch",
                        "parent": "root",
                        "sceneKey": "launch",
                        "title": "Launch",
                        "type": "REACT_NATIVE_ROUTER_FLUX_PUSH"
                      }
                    }
                    onNavigate={[Function bound handleNavigation]}
                    parent="root"
                    sceneKey="launch"
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "borderColor": "red",
                        "borderWidth": 2,
                        "flex": 1,
                        "justifyContent": "center"
                      }
                    }
                    title="Launch"
                    type="REACT_NATIVE_ROUTER_FLUX_PUSH">
                    <Text>
                      Launch page
                    </Text>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Go to Login page
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Go to Register page
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Go to Register page without animation
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Popup error
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Go to TabBar page
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        Go to switcher page
                      </Text>
                    </View>
                    <View
                      accessibilityComponentType="button"
                      accessibilityLabel={undefined}
                      accessibilityTraits="button"
                      accessible={true}
                      hitSlop={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function bound touchableHandleResponderGrant]}
                      onResponderMove={[Function bound touchableHandleResponderMove]}
                      onResponderRelease={[Function bound touchableHandleResponderRelease]}
                      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
                      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
                      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
                      style={
                        Object {
                          "opacity": 1
                        }
                      }
                      testID={undefined}>
                      <Text
                        style={
                          Array [
                            Object {
                              "color": "#007aff",
                              "fontFamily": "HelveticaNeue-Medium",
                              "fontSize": 17,
                              "fontWeight": "bold",
                              "textAlign": "center"
                            },
                            null,
                            undefined,
                            null
                          ]
                        }>
                        back
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
